packages/plugin-ris/src/ris.js

```diff
@@ -48,9 +48,12 @@ export function parse (text) {
   let lastEntry
   let lastTag
 
-  for (let line of text.split('\n')) {
+  for (let line of text.split(/\r?\n/)) {
     line = line.trim()
-    if (!LINE_MATCH.test(line)) { lastEntry[lastTag] += line }
+    if (!LINE_MATCH.test(line)) {
+      if (lastEntry && lastTag) { lastEntry[lastTag] += ' ' + line.trim() }
```

Comment:
Hi, @larsgw. One question: why it's needed to `trim()` line here when `line`  already comes trimed from line `52`?

Reply:
The second one can be removed, the line needs to be trimmed for `LINE_MATCH` mainly.